lua/presence/init.lua

```diff
@@ -346,19 +353,23 @@ function Presence.get_file_extension(path)
 end
 
 -- Get the status text for the current buffer
-function Presence.get_status_text(filename)
+function Presence:get_status_text(filename)
     if vim.bo.modifiable and not vim.bo.readonly then
         if vim.bo.filetype == "gitcommit" then
-            return string.format("Committing changes")
+            return string.format(self.options.git_commit_text, filename)
+        else
+            return string.format(self.options.editing_text, filename)
         end
-        return string.format("Editing %s", filename)
-    elseif file_trees[filename:match "[^%d]+"] then
-        return string.format("Browsing %s", file_trees[filename:match "[^%d]+"])
     else
-        if vim.bo.filetype == "netrw" then
-            return string.format("Browsing Netrw")
+        if file_trees[filename:match "[^%d]+"] then
+            return string.format(self.options.file_tree_text, file_trees[filename:match "[^%d]+"])
```

Comment:
I know this was added in the last PR, but it might be a little cleaner to extract the filename matching here to a variable since it's kinda involved.
```suggestion
        local file_tree = file_trees[filename:match "[^%d]+"]

        if file_tree then
            return string.format(self.options.file_tree_text, file_tree)
```

Reply:
Oop it looks like there a small whitespace issue on L364:
```
Checking lua/lib/log.lua                          OK
Checking lua/presence/discord.lua                 OK
Checking lua/presence/file_assets.lua             OK
Checking lua/presence/file_trees.lua              OK
Checking lua/presence/init.lua                    1 warning

    lua/presence/init.lua:364:1: inconsistent indentation (SPACE followed by TAB)

Checking lua/presence/plugin_managers.lua         OK

Total: 1 warning / 0 errors in 6 files
```

Reply:
Woops, it looks like my nvim config was messed up and wasn't expanding my tabs into spaces. Just fixed!

Reply:
No prob :) Thanks!